### PR TITLE
Details layout

### PR DESCRIPTION
**Pull request by laem**
Implements the layout part of #64

### CODE REVIEW

**Comment by laem** [comment]
@MattiSG I didn't get your comment [here](https://github.com/sgmap/cout-embauche/pull/64/files#r43126060), about changes repeated in 6b861e2

**Comment by MattiSG** [comment]
I can't find the change I commented on in this changeset.

File: css/details.css [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+@media(max-width: 1024px) {
+	table {
+		width: 80%;
+	}
 }
 
 caption {
 	font-weight: bold;
+	border-bottom: 1px solid #C3C3C3;
+	margin-left: 30%;
```
Comment (MattiSG) [inline css/details.css]: Fragile, relies on the current text length.

**Comment by MattiSG** [comment]
GTM

**Comment by laem** [comment]
Was breaking les-aides/embauche integration without  8050acc

**Comment by MattiSG** [comment]
Well, then using a `clear` will break other layouts as well… :wink: 

How did you test on les-aides ?

**Comment by laem** [comment]
@MattiSG  So, it appears that tables indirectly create new [block formatting contexts](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context) :

> Note that display:table does not establish block formatting contexts per se. But because it can generate anonymous boxes, one of them (with display:table-cell) establishes a new block formatting context. In other words, the trigger is the anonymous box, not display:table. 
[source](http://yuiblog.com/blog/2010/05/19/css-101-block-formatting-contexts/)

So as long as we have tables for details, integrations with floated and high elements could be problematic. This clear button won't break anything more at the moment.

**Pull request merged**